metadata/index.html

```diff
@@ -1446,6 +1459,7 @@
           <li>when <strong>displaying</strong> data, to map it into formats that are meaningful for those viewing the data (as opposed to those publishing it)</li>
           <li>when <strong>inputting</strong> data, to turn entered values into representations in a consistent format</li>
         </ul>
+        <p>The <dfn>cell value</dfn> of a cell is the result processing the cell as defined here and applying any format-specific mappings to create on or more values. Converters MUST be provided these cell values, along with any in-scope <code>lang</code> and <code>datatype</code> <a title="property value">property values</a>.</p>
```

Comment:
Maybe we need a different term. In the model document _cell value_ describes the semantic value without metadata. This definition defines the value(s) after using metadata to map those values. I'd say we either update the model document to describe _cell value_ using metadata mapping, or create a new term such as _mapped cell value_ to describe what's defined here. WDYT?

> Aren't converters provided everything from the metadata?

This statement just indicates the an implementation must provide lang and datatype information along with the value for use in the converter. My implementation creates a `Cell` class including everything needed for RDF or JSON to properly perform a transformation, but that detail is not defined here. Wording improvements are welcome.